src/main/java/hyperledger/besu/java/rest/client/EthConnectorApplication.java

```diff
@@ -6,11 +6,13 @@
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+import org.springframework.scheduling.annotation.EnableAsync;
 
 /** Bootstrap for the Eth-Connector application. */
 @Slf4j
 @SpringBootApplication
 @EnableConfigurationProperties
+@EnableAsync
```

Comment:
Why we added this annonation ?

Reply:
switches on Spring's ability to run @Async methods in a background thread pool